### DISCUSSION TITLE
Guide to install cloudforet ?

### DISCUSSION THREAD

**Discussion by gikang82** [cloudforet-io/community#95] [General]
Hello
I would like to introduce this project to my customer about this project, so I am try to install it.
Checking out some documents. It seems like there is two repos about installation

https://github.com/cloudforet-io/charts/tree/master/examples/v1.10.4
https://github.com/cloudforet-io/spaceone-helm#commands

Does anyone can recommand which instruction to follow?

**Comment by choonho** [comment]
spaceone-helm is not maintained. This is deprecated repo.

Please use charts repository.
